GPU/Software/SoftGpu.cpp

```diff
@@ -730,7 +730,18 @@ void SoftGPU::NotifyDisplayResized() {
 	}
 }
 
-void SoftGPU::NotifyConfigChanged() {}
+void SoftGPU::NotifyConfigChanged() {
+	configChanged_ = true;
+}
+
+void SoftGPU::CheckConfigChanged() {
+	if (configChanged_) {
+		drawEngineCommon_->NotifyConfigChanged();
+		BuildReportingInfo();
+		presentation_->UpdatePostShader();
```

Comment:
Should `GPUCommon::CheckConfigChanged()` be moved to GPUCommonHW?  Also, apparently this is crashing from headless now.

-[Unknown]

Reply:
Yeah, that might be a good idea.